### DISCUSSION TITLE
ERROR Unable to initialize gateway backend: The difference between the request time and the server's time is too large..

### DISCUSSION THREAD

**Discussion by mgyboom** [minio/minio#17194] [Q&A] [answered]
```shell
docker run -d -p 9000:9000 --name minio \
  --restart always \
  -e "MINIO_ACCESS_KEY=root" \
  -e "MINIO_SECRET_KEY=r1o2o3t4" \
  minio/minio:RELEASE.2019-08-07T01-59-21Z \
  gateway s3 http://ip:9000
```

docker logs -f minio:
ERROR Unable to initialize gateway backend: The difference between the request time and the server's time is too large..

**Comment by vadmeste** [comment, marked as answer]
@mgyboom This error most likley means you need to fix date/time in your docker machines. Also the gateway is removed since some time.
